packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js

```diff
@@ -89,14 +102,7 @@ class DateTimeView extends React.PureComponent {
 						goToNextMonth={this.goToNextMonth}
 					/>
 				</div>
-				<div className={theme.time}>
-					<TimePicker
-						allowFocus={this.props.allowFocus}
-						value={this.props.selectedTime}
-						onChange={this.props.onSelectTime}
-						useSeconds={this.props.useSeconds}
-					/>
-				</div>
+				{this.getTimePicker()}
```

Comment:
Can you replace that by `this.props.useTime && this.getTimePicker()`. Then in getTimePicker, you don't need the ternary